snafu/dns_perf_wrapper/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+RUN dnf install -y epel-release && dnf install -y --nodocs redis python3 python3-pip openssl-devel ck-devel yum-plugin-copr && \
```

Comment:
Why do we have 3 dnf commands? Can you reorder and put `dnf copr enable` as the top most, and intall all the necessary packages in a single command?

Reply:
Discussed about this in slack. Thanks.